src/main/clojure/clara/rules/schema.cljc

```diff
@@ -1,7 +1,10 @@
 (ns clara.rules.schema
   "Schema definition of Clara data structures using Prismatic's Schema library. This includes structures for rules and queries, as well as the schema
    for the underlying Rete network itself. This can be used by tools or other libraries working with rules."
-  (:require [schema.core :as s]))
+  (:require [schema.core :as s])
+  #?(:clj
+     (:import [clojure.lang
+               IFn])))
```

Comment:
Minor: You could use the [ifn? predicate](https://clojuredocs.org/clojure.core/ifn_q) instead if you want to avoid the need for conditionals.  Looks like that has been in ClojureScript since 2012: [commit](https://github.com/clojure/clojurescript/commit/88a82720bd0642ace4d751343372c3106c8556ea)